vms/administrator/models.py

```diff
@@ -21,7 +21,7 @@ class Administrator(models.Model):
     address = models.CharField(
         max_length=75,
         validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\.)|(,)|(\:)|(@_!#$%^&*()<>?/\|}{~:)]+$', ),
+            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\.)|(,)|(\:)|(/\.,-_)]+$', ),
```

Comment:
Here's a much simple shorter one 
```suggestion
            RegexValidator(r'^[A-Za-z0-9\s\-.,_/:]+$', ),
```